api.go

```diff
@@ -100,7 +99,7 @@ func Server() {
 			}
 
 			// Prepare request object to send files from form to bucket
-			bucketUrl := supabase.BaseURL + "/" + "storage/v1/object/assignments/" + assignmentId + "/"
+			bucketUrl := supabase.BaseURL + "/storage/v1/object/assignments/" + assignmentId + "/"
```

Comment:
nit: may not need the trailing slash

Reply:
oop may be a leftover artifact from the way it previously worked. ill test it

Reply:
wait no this is necessary, used when making requests to the storage bucket
```go
bucketReq, err := http.NewRequest(
	http.MethodPost,
	bucketUrl+studentId+"_"+header.Filename,
	bytes.NewReader(fileBytes),
)
```